packages/create-webstone-app/src/tasks/2-setup-project/plugin.ts

```diff
@@ -0,0 +1,89 @@
+//@ts-ignore this package doesn't provdide a declaration file
+import { create } from "create-svelte";
+import { Ctx, getAppName } from "../../helpers";
+import path, { dirname } from "path";
+import fs from "fs-extra";
+import { fileURLToPath } from "url";
+import { ListrTask } from "listr2";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export const copyTemplate = (ctx: Ctx) => {
+  const appDir = ctx.appDir;
+  fs.copySync(
+    path.join(__dirname, "..", "template", "plugin", "structure"),
+    appDir
+  );
+};
+
+export const renameChildPackage = (ctx: Ctx) => {
```

Comment:
I'd probably make the function name more specific given it can only be used to rename the CLI package.

```suggestion
export const renameCliPackage = (ctx: Ctx) => {
```